### PR TITLE
Bug/user credentials error

### PR DESCRIPTION
**Pull request by elibosley**
# PR for issue #No-Issue .

## Ready for a PR?
 - [x] I have updated the CHANGELOG with an entry including a description, a link to my profile, and a link to the issue ticket. This change is filed under an Enhancement or Bug Fix, is shown under the relevant release tag name, and is included in my PR branch. 
 
## Breaking changes:
None, should unbreak some changes

### CODE REVIEW

**Comment by elibosley** [comment]
@dehli This should be fixed now, let me know if it looks good to you!

File: build.boot [hunk 1]
```diff
@@ -219,8 +219,11 @@ Type \"Yes\" to confirm."
                          (clojure.java.io/file)
                          (slurp)
                          (clojure.edn/read-string))
-        config-settings (load-file
-                         (str (System/getProperty "user.dir") "/config/credentials.edn"))]
+        config-settings (or
+                          (load-file
```
Comment (dehli) [inline build.boot]: I would swap the order the files are loaded so that if you have the file in both locations it reverts to `~/.wombats/config.edn`

Comment (elibosley) [inline build.boot]: Sounds good, I'll fix this and get it back to you!

**Comment by elibosley** [comment]
@dehli Should be better now!

File: build.boot [hunk 2]
```diff
@@ -211,6 +211,14 @@ Type \"Yes\" to confirm."
     (str uri "?aws_access_key_id=" akid "&aws_secret_key=" sk)
     uri))
 
+(defn- get-private-config-file
+  [] (let [file-location-dev (str (System/getProperty "user.dir") "/config/config.edn")
```
Comment (dehli) [inline build.boot]: I would keep `[]` on its own line.

**Pull request merged**